drivers/hmis/README.md

```diff
@@ -1,3 +1,24 @@
-## Hmis README
+## HMIS Driver
 
-This README file should be used to explain the functionality of the driver.
+This driver contains all the backend logic for supporting the [HMIS Frontend](https://github.com/greenriver/hmis-frontend).
+
+To enable locally:
+```
+ENABLE_HMIS_API=true
+HMIS_HOSTNAME=hmis.dev.test
+```
+
+### New Deployment Checklist
+
+Some things need to be done manually for a new deployment:
+
+* Create the HMIS Data Source
+* Create an HMIS Administrator role
+* Configure permissions in the warehouse
+* Set up File Tags in the warehouse
+* Create any UnitTypes
+* Create any CustomServiceTypes and categories
+* Create any CustomDataElementDefinitions
+* Set up any RemoteCredentials
+* Enable any InboundApiConfigurations
+* Create a GrdaWarehouse::Theme
```

Comment:
Is it worth putting in some short code to explain how to do this?

Reply:
I don't think so yet, because I feel like it will just become out-of-date quickly.. referencing the model names should be enough to figure it out. The Theme model has an example for how to created it as a code comment